### PR TITLE
Cleanup whitespace

### PR DESCRIPTION
**Pull request by mfwass**
**Note**: Only Python files were edited.

This merge request adds some whitespace to certain areas that makes it easier to read and also removes some whitespace from areas where it is not needed.

There is also a tab-->spaces conversion in tools/SourceKit/bindings/python/sourcekitd/capi.py

### CODE REVIEW

File: stdlib/public/common/MirrorCommon.py [hunk 1]
```diff
@@ -23,26 +23,35 @@ def getDisposition(disp=None):
...
   for arg in genericArgs:
     if not first:
       genericArgString  = genericArgString + ','
+
     first = False
     genericArgString = genericArgString + arg
+
   if genericConstraints == None:
     genericConstraintString = genericArgString
+
```
Comment (gribozavr) [inline stdlib/public/common/MirrorCommon.py]: I think a newline here is misleading.

File: utils/SwiftIntTypes.py [hunk 2]
```diff
@@ -90,16 +90,16 @@ def numeric_type_names_Macintosh_only():
 
 def all_integer_binary_operator_names():
     return ['<<', '>>', '&*', '&', '&+', '&-', '|', '^']
-
```
Comment (gribozavr) [inline utils/SwiftIntTypes.py]: Please filter out changes that remove trailing whitespace.

File: validation-test/stdlib/Slice/Inputs/GenerateSliceTests.py [hunk 3]
```diff
@@ -5,11 +5,13 @@
 traversal_options = [ 'Forward', 'Bidirectional', 'RandomAccess' ]
 base_kind_options = [ 'Defaulted', 'Minimal' ]
 mutable_options = [ False, True ]
+
 for traversal, base_kind, mutable in itertools.product(
     traversal_options, base_kind_options, mutable_options):
     # Test Slice<Base> and MutableSlice<Base> of various collections using value
     # types as elements.
     wrapper_types = [ 'Slice', 'MutableSlice' ] if mutable else [ 'Slice' ]
+
```
Comment (gribozavr) [inline validation-test/stdlib/Slice/Inputs/GenerateSliceTests.py]: Why is this empty line an improvement?  It separates the definition of the list and the for loop, making it seem like they are unrelated.

File: stdlib/public/common/MirrorCommon.py (hunk 1 above)
Comment (mfwass) [inline stdlib/public/common/MirrorCommon.py]: I'll remove it if you'd like.

File: validation-test/stdlib/Slice/Inputs/GenerateSliceTests.py (hunk 3 above)
Comment (mfwass) [inline validation-test/stdlib/Slice/Inputs/GenerateSliceTests.py]: Preventing everything from being all cluttered together. Spacing it out IMO makes it easier to read.

File: utils/SwiftIntTypes.py (hunk 2 above)
Comment (mfwass) [inline utils/SwiftIntTypes.py]: What's the purpose of the blank spaces with no code after it?

From the current file in the repo:
https://github.com/apple/swift/blob/master/utils/SwiftIntTypes.py#L93

All that's there is space that's not needed.

Comment (gribozavr) [inline utils/SwiftIntTypes.py]: No purpose, but removing it creates churn that not everyone likes.  Its presence is not harmful.

Comment (mfwass) [inline utils/SwiftIntTypes.py]: Just my coding preference, I guess.

There's places with it, and places without it *in the same file*. So by removing it, there is more code consistency.

Comment (gribozavr) [inline utils/SwiftIntTypes.py]: This is not my battle to fight: pull request #275

Comment (mfwass) [inline utils/SwiftIntTypes.py]: Ah, didn't see #275. I'll close this pull request to save the headache.

However in the future I think it'd be nice for all the trailing whitespace to be removed. :P

**Pull request closed without merging**